optuna/pruners/hyperband.py

```diff
@@ -50,6 +50,10 @@ class HyperbandPruner(BasePruner):
         at least :math:`4 \\times 10` pruners are consumed for startup.
```

Comment:
I was thinking that there's no straight forward way for the user to know how many pruners this class holds. Do you think it would be useful to expose the number of brackets as a public API?

Reply:
I think the number of `SuccessiveHalvingPruner`s depends on the internal logic of Hyperband. In most use cases, users don't need to know the quantity, so I think we don't need have `n_brackets` as a public API.

Reply:
Not introduced by this PR but should it not be the following?
```suggestion
        at least :math:`4 \\times 10` trials are consumed for startup.
```